resources/deu.ini

```diff
@@ -1,208 +1,219 @@
 ; Errors that are displayed when the player does something wrong, or the plugin screws up.
 error=Ups! Irgendetwas ist falsch gelaufen!
-notinplot=Du stehst nicht in einem Plot
-notowner=Dieser Plot gehört dir nicht!
+notinplot=Du befindest dich nicht in einem Grundstück.
+notowner=Dieses Grundstück gehört dir nicht.
 
 confirm=confirm
 
 ; The /p help Subcommand
 help.name=help
-help.desc=Zeigt dir das Menü mit allen Befehlen an
-help.usage=/p help [seite]
-help.header=--- Zeige MyPlot Hilfe Seite {%0} of {%1} ---
+help.desc=Zeigt dir das Hilfemenü an.
+help.usage=/p help [Seite]
+help.header=--- Zeige MyPlot Hilfeseite {%0} von {%1} ---
 
 ; The /p addhelper Subcommand
 addhelper.name=addhelper
 addhelper.alias=addh
-addhelper.desc=Füge einen Helfer zu deinem plot hinzu
-addhelper.usage=/p addhelper <spieler>
-addhelper.alreadyone={%0} ist bereits ein Helfer von diesem Plot!
-addhelper.success={%0} wurde zum Helfer von diesem Plot ernannt!
+addhelper.desc=Füge einen Helfer zu deinem Grundstück hinzu.
+addhelper.usage=/p addhelper <Spieler>
+addhelper.alreadyone={%0} ist bereits ein Helfer von diesem Grundstück!
+addhelper.notaplayer=Dieser Spieler ist momentan nicht online.
+addhelper.success={%0} wurde zu einem Helfer von diesem Grundstück ernannt.
 
 ; The /p removehelper Subcommand
 removehelper.name=removehelper
 removehelper.alias=delh
-removehelper.desc=Entferne einen Helfer von deinem Plot
-removehelper.usage=/p removehelper <spieler>
+removehelper.desc=Entferne einen Helfer von deinem Grundstück.
+removehelper.usage=/p removehelper <Spieler>
 removehelper.notone={%0} ist kein Helfer!
-removehelper.success={%0} wurde von deinen Plot-Helfer Liste entfernt!
+removehelper.success={%0} wurde als Helfer entfernt.
 
 ; The /p denyplayer Subcommand
 denyplayer.name=denyplayer
 denyplayer.alias=deny
-denyplayer.desc=Verbiete einem Spieler deinen Plot zu betreten
-denyplayer.usage=/p deny <player>
+denyplayer.desc=Verbiete einem Spieler dein Grundstück zu betreten.
+denyplayer.usage=/p deny <Spieler>
 denyplayer.helper={%0} ist ein Helfer!
-denyplayer.alreadyone={%0} ist es bereits verboten deinen Plot zu betreten!
-denyplayer.notaplayer=Dieser Spieler ist gerade nicht online!
-denyplayer.cannotdeny=Du kannst {%0} nicht verbieten deinen Plot zu betreten!
-denyplayer.attempteddeny={%0} versucht dich von deinem Plot zu sperren!
-denyplayer.success1={%0} darf ab jetzt deinen Plot nicht mehr betreten!
-denyplayer.success2=Du darfst ab jetzt den Plot  {%0};{%1} von {%2} nicht mehr betreten!
+denyplayer.alreadyone={%0} ist es bereits verboten, dein Grundstück zu betreten!
+denyplayer.notaplayer=Dieser Spieler ist momentan nicht online.
+denyplayer.cannotdeny=Du kannst {%0} nicht verbieten, dein Grundstück zu betreten!
+denyplayer.attempteddeny={%0} versucht, dich von deinem Grundstück zu sperren!
+denyplayer.success1={%0} darf dein Grundstück nun nicht mehr betreten.
+denyplayer.success2=Du darfst nicht mehr das Grundstück {%0};{%1} von {%2} betreten!
```

Comment:
Thats what i mentioned to him on some other subcomments. May be fixable by enclosing everything with " "